raiden/blockchain/state.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+from raiden.routing import make_graph
+from raiden.transfer.state import (
+    NettingChannelEndState,
+    NettingChannelState,
+    TokenNetworkGraphState,
+    TokenNetworkState,
+    TransactionExecutionStatus,
+)
+
+
+def get_channel_state(token_address, reveal_timeout, netting_channel_proxy):
+    channel_details = netting_channel_proxy.detail()
+
+    our_state = NettingChannelEndState(
+        channel_details['our_address'],
+        channel_details['our_balance'],
+    )
+    partner_state = NettingChannelEndState(
+        channel_details['partner_address'],
+        channel_details['partner_balance'],
+    )
+
+    identifier = netting_channel_proxy.address
```

Comment:
What should the channel identifier be after all? [Here](https://github.com/raiden-network/spec/blob/master/smart_contracts.rst#open-questions) it's still as an open question. But being the address of the channel is not in any of the options. Am I missing something?

Reply:
It will not be the channel address in the future https://github.com/raiden-network/raiden/issues/1025 . Identifier is just a better name for the future codebase, atm it is